packages/app-scripts-lit-element/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@snowpack/app-scripts-lit-element",
+  "version": "0.3.4",
+  "main": "snowpack.config.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@babel/plugin-proposal-class-properties": "^7.8.3",
```

Comment:
I've seen that there are some dependencies in the other packages that I didn't include, should I add them? I'm talking about: `@babel/cli`, `@babel/core`, `@babel/plugin-syntax-import-meta`, `@babel/preset-env`, `@snowpack/babel-plugin-package-import` and `babel-plugin-macros`